stdlib/regex.gr

```diff
@@ -95,18 +99,30 @@ record RegExBuf {
   config: RegExParserConfig,
 }
 
-let makeRegExBuf = (s) => {
-  {input: s, inputExploded: String.explode(s), cursor: box(0), config: makeRegExParserConfig()}
+let makeRegExBuf = s => {
+  {
+    input: s,
+    inputExploded: String.explode(s),
+    cursor: box(0),
+    config: makeRegExParserConfig(),
+  }
 }
 
 let withConfig = (buf: RegExBuf, config: RegExParserConfig) => {
-  {input: buf.input, inputExploded: buf.inputExploded, cursor: buf.cursor, config: config}
+  {
+    input: buf.input,
+    inputExploded: buf.inputExploded,
+    cursor: buf.cursor,
+    config,
+  }
 }
 
 // Parsing internals for recursive descent
 
 let parseErr = (buf: RegExBuf, msg: String, posShift) => {
-  "Invalid Regular Expression: " ++ msg ++ " (position " ++ toString(unbox(buf.cursor) + posShift) ++ ")"
+  "Invalid Regular Expression: " ++ msg ++ " (position " ++
+  toString(unbox(buf.cursor) + posShift) ++
+  ")"
```

Comment:
Shouldn't this be
```suggestion
  "Invalid Regular Expression: " ++
  msg ++
  " (position " ++
  toString(unbox(buf.cursor) + posShift) ++
  ")"
```
?

Reply:
Well, the formatter aims to make every line as long as it can.  We can add a break after each ++ to see if we can force it though if you want?

Reply:
It is now :) Fixed.